packages/sdk/src/bootstrap.ts

```diff
@@ -65,7 +65,7 @@ export const initSDK = <T extends SDKConfig>(sdkConfig: T): SDKApi<T> => {
         const methodFromExtend = Reflect.get(extend, propKey, receiver);
         const methodFromTarget = Reflect.get(target, propKey, receiver);
 
-        const method = methodFromExtend ?? methodFromTarget;
+        const method = methodFromTarget ?? methodFromExtend;
```

Comment:
Worth to add a comment here, me and @bartoszherba thinks this was unintended before, `extend` should work only for non-existing methods as we have `override`, so this fix is not related with the core of the feature but worth to add.

Reply:
Yes, forgot to comment this line.